packages/carbon-addons-devenv/src/react/addons_samples/Logo/Logo.js

```diff
@@ -60,10 +66,4 @@ Logo.propTypes = {
   size: PropTypes.oneOf(["small", "medium", "large"]),
 };
 
-Logo.defaultProps = {
-  size: "medium",
-  className: undefined,
-  imageName: undefined,
-};
-
```

Comment:
Default props are deprecated in favour of setting the defaults in the constructor, e.g. `size` prop in `const Logo = ({ children, className, imageName, size = "medium", ...other })`.